### PR TITLE
version upgrades for ES7

### PR DESCRIPTION
**Pull request by bcbrockway**
https://gitlab.com/mintel/satoshi/planning/issues/502

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-gem 'fluent-plugin-elasticsearch', '~>2.11.5'
-gem 'fluent-plugin-systemd', '~>1.0.1'
-gem 'fluent-plugin-detect-exceptions', '~>0.0.11'
-gem 'fluent-plugin-prometheus', '~>1.0.1'
-gem 'fluent-plugin-multi-format-parser', '~>1.0.0'
-gem 'fluent-plugin-rewrite-tag-filter', '>=2.0.0'
-gem 'fluent-plugin-record-modifier', '>=1.1.0'
-gem 'fluent-plugin-http-healthcheck', '>=0.1.0'
-gem 'oj', '~>3.6.5'
+gem 'fluentd', '~> 1.7', '>= 1.7.3'
```
Comment (primeroz) [inline Gemfile]: that is a big jump! :) 

did we review the changelog ?

File: Gemfile [hunk 2]
```diff
@@ -1,15 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'fluentd', '<=1.2.4'
-gem 'activesupport', '~>5.2.1'
```
Comment (primeroz) [inline Gemfile]: what happened to this gem ? don't we need it ?

File: Gemfile [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-gem 'fluent-plugin-record-modifier', '>=1.1.0'
-gem 'fluent-plugin-http-healthcheck', '>=0.1.0'
-gem 'oj', '~>3.6.5'
+gem 'fluentd', '~> 1.7', '>= 1.7.3'
+gem 'fluent-plugin-concat', '~> 2.4'
+gem 'fluent-plugin-kubernetes_metadata_filter', '~> 2.4'
+gem 'fluent-plugin-elasticsearch', '~> 3.5', '>= 3.5.5'
+gem 'fluent-plugin-systemd', '~> 1.0', '>= 1.0.2'
+gem 'fluent-plugin-detect-exceptions', '~> 0.0.12'
+gem 'fluent-plugin-prometheus', '~> 1.6', '>= 1.6.1'
```
Comment (primeroz) [inline Gemfile]: * changes in metrics name, our dashboard is broken anyway so no big deal

File: Gemfile (hunk 1 above)
Comment (bcbrockway) [reply Gemfile]: Of course! :)

It looked like there were no breaking changes - only additions and bugfixes. Seems to be running ok on the updated platform.

File: Gemfile (hunk 2 above)
Comment (primeroz) [reply Gemfile]: cool

**Pull request merged**